docs/data-ai/data/visualize.md

```diff
@@ -25,52 +25,152 @@ Create a dashboard visualizing numeric data from components with the **TELEOP**
+1. Assign a title, a unit suffix, and a refresh rate.
+
+#### Time series
+
+The time series widget creates a graph of tabular data. You can add multiple lines to the time series widget to compare multiple readings over the same time period:
+
+{{<imgproc src="/services/data/visualize-widget-time-series.png" resize="1000x" style="width: 500px" class="fill imgzoom shadow" declaredimensions=true alt="A time series widget displaying a live graph of sensor data over time.">}}
+
+To configure the time series widget, define the following attributes for each line in the time series:
+
+1. From the **Resource name** dropdown, choose a sensor you would like to visualize.
+1. From the **Capture method** dropdown, choose a method of data capture (for example **Readings**).
+1. From the **Path** dropdown, choose the field of data that this line should visualize.
+
+Use the other fields to customize the title, unit, duration, and other aspects of your visualization.
+
+The **window method** allows you to aggregate sensor readings over specified time intervals instead of displaying raw data points.
+Select a window method from the following options:
+
+- **None**: shows raw data with the path specified with no aggregation
+- **Count**: shows the number of readings within the window
+- **Average**: calculates the average value throughout the window
+- **Minimum**: shows the minimum value within the window
+- **Maximum**: shows the maximum value within the window
+- **Custom**: shows the result of a custom MQL aggregation pipeline that you define
+
+#### Table
+
+The table widget displays a grid of historic tabular data values. You can display multiple fields simultaneously in a single table. Each row in the table represents a separate historic reading; each column represents a field.
+
+{{<imgproc src="/services/data/visualize-widget-table.png" resize="800x" style="width: 500px" class="fill imgzoom shadow" declaredimensions=true alt="A table widget displaying a grid of sensor readings.">}}
+
+To configure the table widget, define the following attributes:
+
+1. From the **Resource name** dropdown, choose a sensor you would like to visualize.
+1. From the **Capture method** dropdown, choose a method of data capture (for example **Readings**).
+1. From the **Path** dropdown, choose the data that this table should visualize.
+
+Use a custom MQL aggregation pipeline stage (or series of stages) to transform your sensor data into a flat object where each field corresponds to a column in the table. Consider the following sensor data, which contains information about air quality in a field named `readings`:
+
+```json
+"data" {
+ "readings": {
+  "gas_resistance": 114978.66606781945,
+  "temperature": 22.96,
+  "pressure": 1016.18,
+  "humidity": 48.318
+}
+```
+
+To visualize this data in a table with columns titled "gas_resistance", "temperature", "pressure", and "humidity", use the following custom MQL stage:
```

Comment:
```suggestion
To visualize this data in a table with columns titled "gas_resistance," "temperature," "pressure," and "humidity," use the following custom MQL stage:
```
[nit]
This would be different if you were using monospace (which I think you could here?) but if not, normal punctuation convention applies